### ISSUE TITLE
bert.py中的mutiple sample loss貌似有问题？

### ISSUE THREAD

**Issue by himself07** [thunderboom/competation#1]
原：
            if labels is not None:
                if i == 0:
                    loss = compute_loss(out, labels, loss_method=self.loss_method) / self.multi_drop
                else:
                    loss += loss / self.multi_drop
 改成这样？  
            if labels is not None:
                loss += compute_loss(out, labels, loss_method=self.loss_method) / self.multi_drop

**Comment by shishijier** [comment]
我也觉得有问题，不是这样写的
